### DISCUSSION TITLE
Colour Track with Vehicles

### DISCUSSION THREAD

**Discussion by particl3s** [OpenRCT2/OpenRCT2#19290] [Feature requests] [answered]
Vehicle colour remap track sprites in real-time, like a whirlpool of emotions.  
[![c-colour-track-export.gif](https://i.postimg.cc/cZS7spKh/c-colour-track-export.gif)](https://postimg.cc/fk0bTCFM)

**Comment by sjoerddebruin** [comment, marked as answer]
I think this is easily doable with a plugin, one can even connect it with the G forces...

**Comment by ocalhoun6** [comment]
Yeah... This seems best suited for a plugin. Not something that most players would want to use, and it could add a lot of extra complications.

**Comment by ltsSmitty** [comment]
I've worked on some things along this line with plugins, and seems like quite a cool idea. How do you envision it working for multi-colored cars/tracks? Like would the main track colored => main car color, additional track color=> secondary car colored, and supports => tertiary car color?

**Reply by particl3s** [reply]
Over complication of colour theory is a problem in art. Simplicity is the height of sophistication & ideally some range of mistake is a good thing. 

From a computational standard having a 'random' toggle is useful in this situation. As in simply let a machine decide & let a user specify what to keep after.

So colour A can affect track A, B or C(supports).

**Comment by ltsSmitty** [comment]
I made this plugin, shown here. What should it be called? 

https://imgur.com/P989UI9

**Reply by trzejos** [reply]
wet paint

**Reply by particl3s** [reply]
Mintoo ^^

**Comment by ltsSmitty** [comment]
@particl3s v1 of the plugin is live so you can play around with it. significantly enhanced features on the way as well!

https://github.com/ltsSmitty/Wet-Paint-Plugin

download the release [here](https://github.com/ltsSmitty/Wet-Paint-Plugin/releases/tag/0.1.0)

**Reply by particl3s** [reply]
Here's my first test, thanks again for the add-on. 
https://imgur.com/a/YAlfCj8
